internal/resourcepolicies/resource_policies_mgr.go

```diff
@@ -0,0 +1,139 @@
+package resourcepolicies
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kbClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// resPoliciesMeta stored one resource policies and related resource policies configmap creation timestamp
+type resPoliciesMeta struct {
+	policies                *ResourcePolicies
+	configCreationTimestamp metav1.Time
+}
+
+// resPoliciesKey identify one specific resPoliciesMeta with refName and refType
+type resPoliciesKey struct {
```

Comment:
Use UID instead as the key

Reply:
Here I'm using `refName, refType` as the key as `refName` + `refType` could refer to the same name and type of config.
For example, if the user first refers to the policies as `cm0` + `configmap` but later updates the rules in configmap`cm0`.  then the `resPoliciesMeta` in map of`resPolicies map[resPoliciesKey]*resPoliciesMeta` with key `cm0` + `configmap` will be update, so `refName` + `refType` will keep the related resPoliciesMeta up-to-date.

Also the entry of searching policies is through `refName` and `refType` in backup CR in design, and if using UID it's not easy comparing with `refName` and `refType`